### ISSUE TITLE
Free accounts should be able to regenerate publishable tokens

### ISSUE THREAD

**Issue by robiupui** [iexg/IEX-API#1385]
**Is your feature request related to a problem? Please describe.**
I am using the IEX API in an academic setting where I teach students how to call a remote API from their own applications.  In the course of my instruction, my students **will** see **my** API Token as I'm building my demo app.

**Describe the solution you'd like**
I should be able to regenerate that Publishable Token after my demo is over so that I do not risk a student using my token instead of their own.

**Describe alternatives you've considered**
I deleted my Publishable key, thinking that I would be able to create a new one but I cannot (therefore, I had to create a 2nd - unnecessary - account.)

I thought about using the Secret Token (which I _can_ regenerate), but I do not want to take the chance of putting that out there anywhere.

I do a lot of video recording of my instruction so it's important that I can change the key prior to the videos being published.

**Additional context**

**Comment by McQgit** [comment]
@robiupui I have a paid account so maybe it works different. But, can't you use the "New Publishable Token" button to create a new one and just delete the old one(s)?  It's not as convenient as the Regenerate button on the secret token I suppose - but not difficult.

**Comment by robiupui** [comment]
@McQgit Yes, I know I'm using the **free** account so I can't really do more than beg for scraps.  Still, the free account is targeted for my specific use case (academic) and, really, this is just a good security practice - especially for students who are bound to accidentally push keys in repos, etc.

**Comment by dmikester1** [comment]
I also wish I had this possibility.  I'd like to share my Google Sheet with him that includes my key, but be able to create a new key later on down the road so his will no longer work.
